master/getting-started/docker/index.md

```diff
@@ -32,14 +32,27 @@ you need to install and use Calico.
 
 ## Tutorials
 
-#### [Simple Policy]({{site.baseurl}}/{{page.version}}/getting-started/docker/tutorials/simple-policy)
+#### [Security using Calico Profiles]({{site.baseurl}}/{{page.version}}/getting-started/docker/tutorials/security-using-calico-profiles)
 
 The above guide demonstrates Calico connectivity cross host, and how to limit
-that connectivity using simple Calico policy.
+that connectivity using simple Calico profiles.  One profile is created for
+each network and the connectivity is defined as policy on each profile.
 
-#### [Advanced Policy]({{site.baseurl}}/{{page.version}}/getting-started/docker/tutorials/advanced-policy)
+#### [Security using Calico Profiles and Policy]({{site.baseurl}}/{{page.version}}/getting-started/docker/tutorials/security-using-calico-profiles-and-policy)
 
 The above guide digs deeper into advanced policy configurations for workloads.
+There is stil one profile created for each network but now the profiles define
+labels that are added to each container added to the network.  The policy uses
+the labels in selectors to configure connectivity.
+
+#### [Security using Docker Labels and Calico Policy]({{site.baseurl}}/{{page.version}}/getting-started/docker/tutorials/security-using-docker-labels-and-calico-policy)
+
+The above guide demonstrates Calico connectivity cross host, and how to limit
+that connectivity using Calico policy with Docker container's labels.  In this
+walkthrough the profiles are not created for each network but instead the labels
```

Comment:
> the labels on the Docker containers are added to the workloads.

I don't think this will be clear for a new user. I assume you mean it's "added to the workload endpoint", but even that is a technical detail I don't think needs to be disclosed yet.

The only points I think need to be stressed are:

- Doesn't use profiles at all
- Assigns labels to containers at runtime
- Creates policy which applies based off those labels

The above guide demonstrates Calico connectivity cross host without using Profiles at all. Instead, Calico Policies are defined which apply to containers depending on the labels assigned to them at runtime. This allows for policy adjustment at the container level rather than at the network level.